requirements.in

```diff
@@ -0,0 +1,3 @@
+ops ~= 2.5
```

Comment:
Why this specific version? We're at 2.12 now

Reply:
it was added by charmcraft init. I simply followed what it created. Will update accordingly.

Reply:
requirements.txt was properly generated with 2.12 as the resolved version. With that being said should the version qualifier be removed entirely from here or left as was put in place by the `init`?